docs/app/pages/css/css-sections/icons/ux-icons/ux-icons.component.html

```diff
@@ -21,9 +24,12 @@
 </div>
 
 <div class="row">
-    <div class="col-md-4 col-sm-4 icon-demo-col m-b" *ngFor="let icon of icons">
-        <div class="icon-demo">
-            <i class="{{ iconClass }} p-r-xs" [ngClass]="icon.classname"></i> <span>{{ icon.classname }}</span>
-        </div>
+    <div class="col-md-4 col-sm-4 icon-demo-col m-b" *ngFor="let icon of icons; trackBy: trackByFn">
+        <uxd-icon-preview
```

Comment:
I wonder would it be better if the buttons were uniform sizes? They currently are like this:

[![Image from Gyazo](https://i.gyazo.com/ab20bbc7ca89114704f5db7281ce1f00.gif)](https://gyazo.com/ab20bbc7ca89114704f5db7281ce1f00)

Perhaps this would be better?

[![Image from Gyazo](https://i.gyazo.com/d7061e72bc1047cffb727bcc12ac9c16.gif)](https://gyazo.com/d7061e72bc1047cffb727bcc12ac9c16)

@donna-marijne thoughts?

Reply:
Agree, consistent hit box will be easier to use.

Reply:
hover now full width